himin/day3/K번째수.js

```diff
@@ -0,0 +1,12 @@
+function solution(array, commands) {
+    const kthNumber = [];
+      
+    for (let command of commands){
+        const [i, j, k] = command;
+        const sliceArray = array.slice(i - 1, j);
+        const sortArray = sliceArray.sort( (a, b) => a - b);
+        s.push(sortArray[k - 1]);
+    }
```

Comment:
```suggestion
    for (const command of commands){
        const [i, j, k] = command;
        const sliceArray = array.slice(i - 1, j);
        const sortArray = sliceArray.sort((a, b) => a - b);
        s.push(sortArray[k - 1]);
    }
```
for of문 문에서 command도 값이 변하지 않기 때문에 `const`로 정의하는게 좋습니다

Reply:
@yihimin 코멘트 확인 전에 머지가 되었네요ㅠ 다음부터는 const로 정의하겠습니다!